packages/db/tests/translate/test-pdrs.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const test = require('ava');
+
+const { translatePostgresPdrToApiPdr } = require('../../dist/translate/pdrs');
+
+test('translatePostgresPdrToApiPdr translates postgres record to PDR record', async (t) => {
```

Comment:
I think we need to add a test for when there is no execution, because I think that test would currently fail

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/8ba0c6ae4c1b80cdfd5e79079bae29ddcd64d26b